test/index.test.js

```diff
@@ -1,8 +1,14 @@
-const index = require('..');
 const assume = require('assume');
 const request = require('request-promise-native');
+const thenify = require('tinythen');
+
+const index = require('..');
 const { address } = require('./util');
 
+// Need to set some values for these so localstack works in Travis
+process.env.AWS_ACCESS_KEY_ID = 'foobar';
+process.env.AWS_SECRET_ACCESS_KEY = 'foobar';
```

Comment:
Can these just go in a `/test/setup.js` and require that from the test script, rather than duplicate this across all the files?